AMW_angular/io/src/app/settings/environments/environment-edit/environment-edit.component.ts

```diff
@@ -31,25 +31,31 @@ export class EnvironmentEditComponent {
 
   getTitle(): string {
     if (!this.environment) return;
-    return this.environment.id ? `Edit ${this.getContextType()}` : `Add ${this.getContextType()}`;
+    return this.environment.id
+      ? `Edit ${this.isDomain() ? 'domain' : 'environment'}`
```

Comment:
It's maybe easier to read if `environment.name` is in the getTitle()-Function instead of adding it in the html-template?
```
? `Edit ${this.isDomain() ? 'domain' : 'environment'} ${this.environment.name}`
```